html/captive-portal/templates/provisioner/windows.html

```diff
@@ -5,7 +5,7 @@
           <img src="/content/images/unlock.png" alt="Access granted" />
           <p>[% i18n("In order to complete your connection to the secure SSID, you will need to run the agent available below.") %]</p>
           <p>
-          <a href="/content/packetfence-windows-agent.exe">[% i18n("Click to download the agent.") %]</a>
+          <a href="[% filesid %]">[% i18n("Click to download the agent.") %]</a>
```

Comment:
Rename this variable. When I read it, it could be file_sid or files_id or file_with_dinde_id